sky130/openlane/common_pdn.tcl

```diff
@@ -7,7 +7,7 @@ set ::macro_blockage_layer_list "li1 met1 met2 met3 met4 met5"
 pdngen::specify_grid stdcell {
     name grid
     rails {
-	    met1 {width 0.48 pitch $::env(PLACE_SITE_HEIGHT) offset 0}
+	    met1 {width $::env(PDN_RAIL_WIDTH) pitch $::env(PLACE_SITE_HEIGHT) offset 0}
     }
     straps {
 	    met4 {width 1.6 pitch $::env(FP_PDN_VPITCH) offset $::env(FP_PDN_VOFFSET)}
```

Comment:
There's another one right here- how about making this configurable too?

Reply:
I guess that's the wise thing to do. I'll PR that now.